webpack/devserver.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(webpackConfig) {
         chunks: false,
         chunkModules: false
       }
-    }).listen(8888, 'localhost', function(err) {
+    }).listen(8888, '0.0.0.0', function(err) {
```

Comment:
> Can you please add link to issue?

You mean as a code comment?

> we should ask Dan Abramov, why hot reload isn't working within VM I suppose.

I think it's a problem of socket forwarding (webpack-dev-server), not related to react hot loader.